components/brave_ads/core/internal/ads/serving/choose/predict_ad_embeddings.h

```diff
@@ -0,0 +1,57 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_COMPONENTS_BRAVE_ADS_CORE_INTERNAL_ADS_SERVING_CHOOSE_PREDICT_AD_EMBEDDINGS_H_
+#define BRAVE_COMPONENTS_BRAVE_ADS_CORE_INTERNAL_ADS_SERVING_CHOOSE_PREDICT_AD_EMBEDDINGS_H_
+
+#include <vector>
+
+#include "base/check_op.h"
```

Comment:
Should be `check.h` because we are not using `DCHECK_EQ` etc.

Reply:
Please remove `check_op.h` as we should use `check.h` if we do not use operator checks

Reply:
we are using `DCHECK_EQ`

Reply:
if we use `DCHECK` or `CHECK` then we should include check.h. We should only include check_op.h if we only use operator checks